### ISSUE TITLE
Building on TravisCI/Linux, can't find swift-build

### ISSUE THREAD

**Issue by uraimo** [kylef/swiftenv#34]
Hi Kyle, at the moment i'm using the [following .travis.yml](https://github.com/uraimo/SwiftyGPIO/blob/master/.travis.yml), a variation on the sample from the docs:

```
os:
  - linux
language: generic
sudo: required
dist: trusty
osx_image: xcode7.2
install:
  - eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/9f442512a46d7a2af7b850d65a7e9bd31edfb09b/swiftenv-install.sh)"
env:
  - SWIFT_VERSION=2.2.1-SNAPSHOT-2016-04-23-a
script:
  - swift build
```

I tried with the `SWIFT_VERSION` you see above and with `2.2`, but the result is always the same, travis can't find the `swift-build` binary:

```
Setting environment variables from .travis.yml
$ export SWIFT_VERSION=2.2.1-SNAPSHOT-2016-04-23-a

$ bash -c 'echo $BASH_VERSION'
4.3.11(1)-release
$ eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/9f442512a46d7a2af7b850d65a7e9bd31edfb09b/swiftenv-install.sh)"
Cloning into '/home/travis/.swiftenv'...
remote: Counting objects: 67, done.
remote: Compressing objects: 100% (57/57), done.
remote: Total 67 (delta 4), reused 34 (delta 3), pack-reused 0
Unpacking objects: 100% (67/67), done.
Checking connectivity... done.
Downloading https://swift.org/builds/swift-2.2-branch/ubuntu1404/swift-2.2.1-SNAPSHOT-2016-04-23-a/swift-2.2.1-SNAPSHOT-2016-04-23-a-ubuntu14.04.tar.gz
2.2.1-SNAPSHOT-2016-04-23-a has been installed.

$ swift build
error: unable to invoke subcommand: /home/travis/.swiftenv/versions/2.2.1-SNAPSHOT-2016-04-23-a/usr/bin/swift-build (No such file or directory)

The command "swift build" exited with 2.
```

I'm a missing something obvious in my `travis.yml`?

Full log [here](https://api.travis-ci.org/jobs/135613583/log.txt?deansi=true), everything else seems to be working.

**Comment by kylef** [comment]
Hi @uraimo, there isn't a `swift build` command included in Swift 2.2/2.2.1. Se Apple's announcement blog post for 2.2 (https://swift.org/blog/swift-2-2-released/).

> The Swift Package Manager is still early in development and is not included in this release.

If you're not doing anything complicated, I've actually written a separate build tool that you might be able to use as a dropin replacement for Swift build that works on Swift 2. https://github.com/kylef/swim.

**Comment by uraimo** [comment]
Oh, thanks, as expected it was something obvious :)
